coffee/chosen.jquery.coffee

```diff
@@ -168,6 +169,13 @@ class Chosen extends AbstractChosen
     @container.addClass "chosen-container-active"
     @active_field = true
 
+    windowHeight = $ window.height() + $ 'html'.scrollTop()
```

Comment:
@lode You should include the brackets in this case. `window.height()` is being used instead of `$(window).height()` hence the error in Travis.
```coffee
windowHeight = $(window).height() + $('html').scrollTop()
```

You're most likely getting the issue you mentioned in your comment because this actually compiles to $() enclosing the rest of the line, not just up to the dot, as I imagine you're expecting.